js/clinicalTimeline.js

```diff
@@ -872,36 +903,86 @@ window.clinicalTimeline = (function(){
     return Math.max(a, b) - Math.min(a, b);
   }
 
+  function roundUp(time, zoomLevel) {
```

Comment:
I think it's better to make this function more general. Use the same function signature as http://stackoverflow.com/questions/3407012. I don't think it's necessary to have a time and zoomLevel since you already have that information inside `getTickValues`. All you want is to round to a nearest multiple.

For 10 days for instance you could call it like
```
roundDown(minTime.inDays(), 10)
```
Make sure it works on both positive and negative numbers. If the number is negative for instance, you can probably just call the `roundUp` from `roundDown` with the negative number multiplied by -1. E.g.:
```
if (numToRound) < 0 {
  return -1 * roundUp(-1 * numToRound);
}
```